kuma/static/js/wiki-compat-trigger.js

```diff
@@ -32,6 +32,20 @@
                 cache: true
             }).then(function() {
                 $('.bc-table').mozCompatTable();
+
+                if (waffle.flag_is_active('bc-signals')) {
+                    // Load signaling feature
+                    var compatSignalJS = mdn.assets.js['wiki-compat-signal'][0];
+
+                    $.ajax({
+                        url: compatSignalJS,
+                        dataType: 'script',
+                        cache: true
+                    }).fail(function(jqXHR, textStatus, errorThrown) {
```

Comment:
This fails to load when I enable our CSP middleware since it requires the addition of `'unsafe-eval'` to our CSP policy for `script-src`. Is there another way we could do this to avoid that? @schalkneethling @davidflanagan

Reply:
This file is already doing the very same thing above at line 31, so it doesn't seem fair to ask John to avoid it here.  I'm not sure what jQuery is doing here. We might have better CSP luck just injecting a script tag. That might be a different CSP violation, I suppose...

Reply:
Most pages have compat tables now, so this entire wiki-compat-trigger file isn't buying us much. It might be easier to just load the required CSS and JS unconditionally, which would avoid the CSP and $.ajax issue entirely.  For now, I guess I'd just leave it as is.